sycl/source/CMakeLists.txt

```diff
@@ -82,10 +82,6 @@ function(add_sycl_rt_library LIB_NAME)
       $<$<BOOL:${SYCL_BUILD_PI_CUDA}>:pi_cuda>
   )
 
-  target_compile_definitions(${LIB_OBJ_NAME}
-    PUBLIC
-      $<$<BOOL:${SYCL_BUILD_PI_CUDA}>:USE_PI_CUDA>)
```

Comment:
Support of this variable setting in `configure.py`/`compile.py` should be removed also, isn't it?

Reply:
not exactly. We still have separate CUDA build which adds full CUDA support.
The change removes assert when non-CUDA SYCL RT is used by SYCL application build with CUDA target because it may be valid configuration.